vignettes/graph_gof_showcase.Rmd

```diff
@@ -16,20 +16,38 @@ This notebook provides a brief introduction to the early version of the **GoF**
 
```

Comment:
The introduction section is very brief and could still leave the user questioning what the purpose is. Clarify that this notebook guides users through analyzing GoF patterns in external Java projects. It would probably be beneficial to explain GoF/Design Patterns and what they are/why we use them? Why would we want to analyze these patterns and how would the information be beneficial to the user?